app/assets/javascripts/application.js

```diff
@@ -17,6 +17,11 @@
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require controllers/deployment/wizard-module
```

Comment:
Yeah, I think you just need to call `wizard-module.js` simply `wizard.js` - that way, it should find it before the `wizard/` directory.